pages/index.tsx

```diff
@@ -1,25 +1,441 @@
+              <code className="language-markup">sp=25</code>
+            </li>
+            <li>
+              <code className="language-markup">sp=all</code>
+            </li>
+          </ul>
+          <h4>
+            se <span className={styles.badge}>Optional</span>
+          </h4>
+          <p>
+            Filter feed items to a specific season. Requires{' '}
+            <code className="language-markup">uf</code> and{' '}
+            <code className="language-markup">sp</code> parameters.
+          </p>
+          <h6>Example:</h6>
+          <pre>
+            <code className="language-markup">se=2</code>
+          </pre>
+          <h4>
+            ct <span className={styles.badge}>Optional</span>
+          </h4>
+          <p>
+            Filter feed items to a specific category. Requires{' '}
+            <code className="language-markup">uf</code> and{' '}
+            <code className="language-markup">sp</code> parameters.
+          </p>
+          <h6>Example:</h6>
+          <pre>
+            <code className="language-markup">ct=entertainment</code>
+          </pre>
+          <h3>
+            <a href="#audio-parameters" id="audio-parameters">
+              Audio File Sourced Embed Players
+            </a>
+          </h3>
+          <h4>ua</h4>
+          <p>
+            File to play in the embed. Must be a complete URL to an audio file.
+            When used with the <code className="language-markup">uf</code>{' '}
+            parameter, this file will play instead of the audio of the RSS item.
+            Has no effect when used with{' '}
+            <code className="language-markup">sp</code> parameter.
+          </p>
+          <h6>Example:</h6>
+          <pre>
+            <code className="language-markup">
+              ua=https://example.com/files/audio.mp3
+            </code>
+          </pre>
+          <h4>tt</h4>
+          <p>Title to display in player.</p>
+
+          <h6>Example:</h6>
+          <pre>
+            <code className="language-markup">tt=My+Audio+Title</code>
+          </pre>
+
+          <h4>ts</h4>
+          <p>Subtitle to display in player.</p>
+
+          <h6>Example:</h6>
+          <pre>
+            <code className="language-markup">
+              ts=Something+About+The+Audio
+            </code>
+          </pre>
+          <h4>ue</h4>
+          <p>Thumbnail image URL. Must be a complete URL to an image file.</p>
+
+          <h6>Example:</h6>
+          <pre>
+            <code className="language-markup">
+              ue=https://example.com/files/thumbnail.jpg
+            </code>
+          </pre>
+
+          <h3>
+            <a href="#customization-parameters" id="customization-parameters">
+              Customization Parameters
+            </a>
+          </h3>
+          <p>
+            These parameters are supported by both the RSS Feed and Audio File
+            embed players. All parameters are optional.
+          </p>
+          <h4>ui</h4>
```

Comment:
Should this be `ui` or `ue`? the example below is `ue`

Reply:
The example is incorrect. It should use the `ui` parameter.